Library/Homebrew/dev-cmd/contributions.rb

```diff
@@ -0,0 +1,109 @@
+# typed: true
+# frozen_string_literal: true
+
+require "cli/parser"
+
+module Homebrew
+  extend T::Sig
+
+  module_function
+
+  SUPPORTED_REPOS = [
+    %w[brew core cask],
+    OFFICIAL_CMD_TAPS.keys.map { |t| t.delete_prefix("homebrew/") },
+    OFFICIAL_CASK_TAPS.reject { |t| t == "cask" },
+  ].flatten.freeze
+
+  sig { returns(CLI::Parser) }
+  def contributions_args
+    Homebrew::CLI::Parser.new do
+      usage_banner "`contributions` <email|name>"
```

Comment:
How about this:

```suggestion
      usage_banner "`contributions` <email|name> [<--repositories>`=`]"
```

It's a little funky, but I think it makes the usage string more consistent with the auto-generated ones (e.g. see `missing`)